torch/_inductor/fx_passes/quantization.py

```diff
@@ -587,12 +592,6 @@ def fn(match):
             and isinstance(binary_node_inputs[1].meta.get("val", None), torch.Tensor)  # type: ignore[union-attr]
         ):
             return False
-        # the two inputs of binary node should have the same shape
```

Comment:
Shouldn't we check the size of binary node is able to be broadcasted instead of removing this check?

Reply:
I think it may not be necessary, because all cases of add are now supported.